ansible/_lint_roles.yml

```diff
@@ -0,0 +1,21 @@
+---
+##
+## This playbook is only used by ansible-lint to test all roles
+##
+## TODO: uncomment roles once the syntax check runs through or document a reason
```

Comment:
```suggestion
## If a role shouldn't be checked, add it commented-out and document a reason
```

Reply:
Yes you are right. I copied that from another repo where this wording makes sense but in our repo it doesn't. Will fix.